notebooks/official/migration/UJ7 Vertex SDK AutoML Text Entity Extraction.ipynb

```diff
@@ -43,10 +43,58 @@
         "      View on GitHub\n",
```

Comment:
<p>you can keep this pip install separate -- as-is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1057/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>